app/services/github_scrape_service.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+class GithubScrapeService
+  SCENARIOS_DIR = './app/scenarios'
+  RAW_FILE_DIR = 'https://raw.githubusercontent.com/ServiceInnovationLab/openfisca-aotearoa/master/'
```

Comment:
> can this be in config? or maybe the branch in config too.

Reply:
@Br3nda when you mean config, do you mean like `config/development.rb?` or in `.env`